site/ar_LB.all.json

```diff
@@ -1394,5 +1394,21 @@
   },
   "usersubscriptions_subscription_status_cancelling": {
     "other": "الإلغاء"
+  },
+  "interval_day": {
+    "one": "يوم",
```

Comment:
These aren't going to pass (future) api validations.  If a translation is un-Count-able, it should only have 'other'.  If it is Count-able, it needs to have every plurality for that language e.g. [ar has all pluralities](https://github.com/nicksnyder/go-i18n/blob/57bd175f4eca85b370ebc664ec389a6c40d89d81/v2/internal/plural/codegen/plurals.xml#L217).

Reply:
What's best practice for 'zero' which would be an impossible situation?

Reply:
That new commit isn't quite right... the languages don't all have the same pluralities... you have to go through that XML and pick the specific pluralities for each language.  And yeah you have to include 'zero' for languages that require it even if the code prevents it from being used because from a translation perspective, it's not an impossible situation.

Sorry it's so annoying lol.

Reply:
I think I've committed the pluralities for each lang correctly but safety check reqd. please and thank you